packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -432,6 +437,38 @@ function TldrawEditorWithReadyStore({
 		() => editor?.getCrashingError() ?? null
 	)
 
+	// For our examples site, we want autoFocus to be true on the examples site, but not
+	// when embedded in our docs site. If present, the `preserveFocus` search param
+	// overrides the `autoFocus` prop and prevents the editor from focusing immediately,
+	// however here we also add some logic to focus the editor when the user clicks
+	// on it and unfocus it when the user clicks away from it.
+	useEffect(
+		function handleFocusOnPointerDownForPreserveFocusMode() {
+			if (!editor) return
+
+			function handleFocusOnPointerDown() {
+				if (!editor) return
+				editor.focus()
+			}
+
+			function handleBlurOnPointerDown() {
+				if (!editor) return
+				editor.blur()
+			}
+
+			if (autoFocus && noAutoFocus()) {
+				editor.getContainer().addEventListener('pointerdown', handleFocusOnPointerDown)
```

Comment:
remind me why we didn't want something like this to be built-in? also, don't we already have something that does the blurring part of this?

Reply:
double checking

Reply:
We have a test that seems to suggest that it's possible, though it doesn't work in practice. If I comment out these items then the editor never gains / loses focus.

Reply:
Oh, there is a test that checks this, but it also sets up the events that make it work. It doesn't / shouldn't work normally! I wish it could but it can't.